reporter/src/funTest/assets/spdx-document-reporter-expected-output.spdx.json

```diff
@@ -0,0 +1,14 @@
+{
+  "SPDXID" : "SPDXRef-<REPLACE_UUID>",
+  "spdxVersion" : "SPDX-2.2.1",
```

Comment:
The format for the SPDX version field is `SPDX-M.N` -- the major and minor versions are included, but not the point releases if there is one -- see https://spdx.github.io/spdx-spec/v2-draft/document-creation-information/#61-spdx-version-field

So this should just be `"SPDX-2.2"`, even after the 2.2.1 release of the spec comes out. The intention is that the point release 2.2.1 is not changing anything substantive in the spec, so SPDX 2.2.1 documents should be identical to 2.2.